packages/browser-sdk/src/stories/components/VideoExperience.tsx

```diff
@@ -120,6 +123,37 @@ export default function VideoExperience({
         });
     }
 
+    function showRequestVideoEnableNotification({ message }: RequestVideoEvent) {
+        toast(
+            (t) => (
+                <div>
+                    {message}
+                    <div>
+                        <button
+                            onClick={() => {
+                                toggleCamera(true);
```

Comment:
With the SDK camera off, this action did not work for me when I requested the SDK user to turn on their camera from a PWA host URL. I did see the toast notification correctly in the SDK but clicking on `Start video` button only resulted in the following error in the console: `Action cannot be performed outside of a connected room`

Reply:
oh. nice catch. I've just tested now and it's actually broken for microphone as well. Need to dig a bit deeper to see whats going on.

Reply:
This is now fixed in the last commit by moving the `roomConnection.status === "connected"` check in to the `core` package (away from `useRoomConnection` ⭐